resources/js/main.js

```diff
@@ -143,15 +144,16 @@ function addEventListenerToButton() {
 // Adds a new item to the todo list
 function addItemToDOM(text, completed) {
 
-  var list = (completed) ? document.getElementById('completed') : document.getElementById('todo');
-  let liWithButtonsElement = createLiWithButtons(text);
-  // let buttonElement = liWithButtonsElement.buttonElements;
+  let list = (completed) ? document.getElementById('completed') : document.getElementById('todo');
+  let aToDoLiElement = createLiElement(text);
```

Comment:
rename to toDoElement, normally do not use `a` in naming, try not use html tag name in naming, such as `li`